packit_service/worker/handlers/fedmsg_handlers.py

```diff
@@ -143,6 +143,7 @@ def run(self) -> HandlerResults:
 
 @add_topic
 @use_for(job_type=JobType.copr_build)
+@use_for(job_type=JobType.build)
```

Comment:
So this will solve #650?

Reply:
OK, that's a different problem.

Reply:
tests brought this up - jobs were not being matched b/c of JobType.build vs. JobType.copr_build